### ISSUE TITLE
fn stuck after locking screen

### ISSUE THREAD

**Issue by HeyItsGilbert** [tekezo/Karabiner-Elements#277]
After locking my screen I noticed that the `fn` key gets stuck. I brought up the on screen keyboard to verify (screenshot below). The only way to reset this is to disconnect and reconnect the keyboard (or reboot the laptop).

![2016-10-10_10-22-12](https://cloud.githubusercontent.com/assets/615265/19244939/8266d2d6-8ed3-11e6-84e0-fa09375819ef.png)

.karabiner.d/configuration/karabiner.json:

``` json
{  "profiles": [
    {
      "name": "Default profile",
      "selected": true,
      "simple_modifications": {
        "caps_lock": "f13",
        "application": "fn"
      }
    }
  ]
}
```

This is on sierra 10.12 (16A323) running Karabiner-Elements v0.90.46 using my WASDkeyboard v2 with only the Mac dip switch on. Happy to provide any logs.

**Comment by HeyItsGilbert** [comment]
Just had this happen with v.90.v47 but just upgraded to 49. I'll report if it occurs again.

**Comment by HeyItsGilbert** [comment]
Still happening on v.0.90.64. Seems to happen when I hit a ton of keys at once (imagine slamming the home row a few times).

**Comment by cremacs** [comment]
I also have this problem, but it happens whenever I lock the screen (i.e. doesn't seem to be because of the cause mentioned in previous issue).

**Comment by HeyItsGilbert** [comment]
Sorry, I have totally failed to mention this at all but this always occurs after I lock my screen. I usually slam a bunch of keys (because I'm an idiot) to wake my screen up.

**Comment by n8henrie** [comment]
I have a similar issue but with an unknown key (it gets "starred out" on my lock screen password). Makes it so I'm unable to log in, because some random key is repeating infinitely and prevents me from typing a correct password. I have to SSH in and `sudo pkill -9 -i karabiner` to be able to log in.

**Comment by perlun** [comment]
Have also seen this. It's one of the more annoying bugs in Karabiner Elements right now.

**Comment by n8henrie** [comment]
I suffer from something like this, but I can't tell what key is getting stuck. It prevents me from being able to log in. I've tried a number of workarounds like pressing all of the keys (hoping that either one is physically stuck or that hitting it triggers the glitch to realize the key is not actually pressed), changing the keyboard layout (the mouse can still access this menu in the menu bar), and connecting an external keyboard, but I am still unable to log in, presumably due to the ghost key getting pressed.

The only workarounds I've discovered so far:

1. Hold down the power key for 10+ seconds for a hard shutdown
2. Log in via SSH and `sudo pkill -f karabiner`, wait 10 seconds, log in normally
